packages/web/src/components/edit-track/EditTrackForm.tsx

```diff
@@ -220,6 +221,8 @@ const TrackEditForm = (
         })
       )
     }
+
+    togglePreview(track.preview, trackIdx)
```

Comment:
Just wondering about moving this

Reply:
not directly related to this, but I didn't want a situation where the toggle changed the isPreviewPlaying and the event was tracks on the wrong interval.